src/include/sof/lib/clk.h

```diff
@@ -38,6 +38,7 @@ struct clock_info {
 	const struct freq_table *freqs;
 	uint32_t default_freq_idx;
 	uint32_t current_freq_idx;
+	uint32_t waiti_freq_idx;	/* lowest possible clock for waiti stage */
```

Comment:
@abonislawski better to change the naming to e.g. lowest_freq_idx to avoid confusion, as we actually use LPRO for the waiti context.

Reply:
@keyonjie updated with lowest_freq_idx